README.md

```diff
@@ -39,3 +39,13 @@ Floworky uses flow for static type checking, which can be manually run using `np
 
 ### Resources
 See [RESOURCES.md](RESOURCES.md)
+
+### Migrations
```

Comment:
This doesn't feel like the right section for this...  We may need to introduce a new file that lists developer specific information, or add this into an appropriate section in CONTRIBUTING.md